components/brave_wallet/browser/swap_service.cc

```diff
@@ -53,15 +55,48 @@ bool IsMainnetEVMNetworkSupported(const std::string& chain_id) {
           chain_id == brave_wallet::mojom::kArbitrumMainnetChainId);
 }
 
-bool IsEVMNetworkSupported(const std::string& chain_id) {
-  return (chain_id == brave_wallet::mojom::kGoerliChainId ||
-          IsMainnetEVMNetworkSupported(chain_id));
-}
-
 bool IsSolanaNetworkSupported(const std::string& chain_id) {
   return chain_id == brave_wallet::mojom::kSolanaMainnet;
 }
 
+bool HasRFQTLiquidity(const std::string& chain_id) {
+  return (chain_id == brave_wallet::mojom::kMainnetChainId ||
+          chain_id == brave_wallet::mojom::kPolygonMainnetChainId);
+}
+
+std::string GetFee(const std::string& chain_id) {
+  std::string fee;
+  if (IsEVMNetworkSupported(chain_id)) {
+    fee = brave_wallet::kBuyTokenPercentageFee;
+  } else if (IsSolanaNetworkSupported(chain_id)) {
+    fee = brave_wallet::kSolanaBuyTokenFeeBps;
+  }
+
+  return fee;
+}
+
+std::string GetFeeRecipient(const std::string& chain_id) {
+  std::string feeRecipient;
+
+  if (IsEVMNetworkSupported(chain_id)) {
+    feeRecipient = brave_wallet::kEVMFeeRecipient;
+  } else if (IsSolanaNetworkSupported(chain_id)) {
+    feeRecipient = brave_wallet::kSolanaFeeRecipient;
+  }
+
+  return feeRecipient;
+}
+
+std::string GetAffiliateAddress(const std::string& chain_id) {
+  std::string affiliateAddress;
+
+  if (IsEVMNetworkSupported(chain_id)) {
+    affiliateAddress = brave_wallet::kAffiliateAddress;
+  }
+
+  return affiliateAddress;
+}
+
```

Comment:
I think early return increases readability a bit.
```
std::string GetFee(const std::string& chain_id) {
  if (IsEVMNetworkSupported(chain_id)) {
    return brave_wallet::kBuyTokenPercentageFee;
  } else if (IsSolanaNetworkSupported(chain_id)) {
    return brave_wallet::kSolanaBuyTokenFeeBps;
  }

  return "";
}
```
and so on

Reply:
Fixed in 650ad4c6ba1e2862d98f14163fae883d828443c9.